angular/wolox-books/src/app/components/services/user/user.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { WoloxBooksApi } from '../../constants/app.constant';
+import { User } from '../../models/user.model';
+import { Observable } from 'rxjs';
+
+import 'rxjs/add/operator/map';
+import 'rxjs/add/observable/of';
+
+@Injectable()
+export class UserService {
+  constructor(private http: HttpClient) {
+    this.http = http;
```

Comment:
you don't need to add this line, this is automatically done with all variables received and not used in the constructor. Just leave the method empty